bloom-housing/bloom
# 1445/open house

# Pull Request Template

## Issue
Closes: #1445

Addresses # (issue)


## Description

This PR adds new elements to the "Application Dates" section. A user has the ability to define event details.

## Type of change


## How Can This Be Tested/Reviewed?

1. Navigate to the partners app
2. Fill open house form
3. Save and see the table on the details page



## Checklist:
